### PR TITLE
Add Form and footer

### PR DESCRIPTION
**Pull request by acolombo1**
Add form-section and footer, and style all to look like the figma design

### CODE REVIEW

**Review by prolajumokeoni — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -194,6 +194,38 @@ <h1>Skills</h1>
...
+      <ul> 
+        <li>
+          <input type="text" id="full_name" name="full_name" maxlength="30" placeholder="Full name" required>
+        </li>
+        <li>
+          <input type="email" id="email" name="user_email" maxlength="30" placeholder="Email address" required>
+        </li>
+        <li>
+          <textarea id="message" name="user_message" maxlength="500" placeholder="Enter text here"></textarea>
+        </li>
```
Comment (AmaduKamara) [inline index.html]: - [x] Kindly add the `required` attribute on the `textarea` field to stop the user from sending an empty message for that field. This attribute is there to activate the validation upon submission of the form. For this project, you were required to have it set. Please fix that. 👍

File: style.css [hunk 2]
```diff
@@ -437,3 +449,103 @@ button.resume:visited {
   background: #36b37f;
   color: #fff;
 }
+
+.form_section {
+  padding: 0 24px;
+}
```
Comment (AmaduKamara) [inline style.css]: - [x] Please add the background image for this section to match your design with the Figma one
- Your Design
image

- The Figma Design
image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️

Hi Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

SoCloseKeeganMichaelKeyGIF


## Highlights
✔️ Linters are running fine
✔️ README is professional
✔️ Good Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
## Hello @acolombo1 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**